CHANGES.md

```diff
@@ -1,3 +1,21 @@
+# 1.2.0
+
+New features:
+* Introduce Object Sizes (GH-808).
```

Comment:
Introduced (note the "d"). 

In general I'm thinking we should start linking to documentation with new features.